lib/gui/app/modules/progress-status.js

```diff
@@ -37,25 +40,26 @@ const utils = require('../../../shared/utils')
  * // '55% Flashing'
  */
 exports.fromFlashState = (state) => {
-  const isChecking = state.type === 'check'
+  const isFlashing = Boolean(state.flashing)
```

Comment:
Perhaps something like `const anyFlashing = (state.flashing >= 1)` would be more explicit?

Reply:
Not preferred by the linter though, iirc